libbpf-tools/klockstat.c

```diff
@@ -64,12 +64,12 @@ static struct prog_env {
 	.iterations = 99999999,
 };
 
-const char *argp_program_version = "klockstat 0.1";
+const char *argp_program_version = "klockstat 0.2";
 const char *argp_program_bug_address =
 	"https://github.com/iovisor/bcc/tree/master/libbpf-tools";
 static const char args_doc[] = "FUNCTION";
 static const char program_doc[] =
-"Trace mutex lock acquisition and hold times, in nsec\n"
+"Trace mutex/rwsem lock acquisition and hold times, in nsec\n"
```

Comment:
optional: i imagine the next thing someone will add is generic semaphore support, so maybe rename "rwsem" with "sem'?  otherwise we're hoping whoever adds full semaphores will remember to change this text.

Reply:
Sure, I'll add the comment and rename to "sem".